src/setup.cpp

```diff
@@ -69,12 +69,12 @@ static void parse_jb_setup(void);
 /*! \brief Play sound effects / music if adjusting it */
 static void sound_feedback(int val)
 {
-    Music.set_volume(val * 10, 0);
+    Music.set_volume(val * 10);
```

Comment:
Do you know what this magic-number `10` represents?

Reply:
I see a `10` on this line that we're dividing out:
```c++
    p = getavalue(_("Sound Volume"), 0, 25, gsvol / 10, true, sound_feedback);
```
I'm not familiar enough to see what it's trying to do, though.

Reply:
When you change the volume in the config screen it represents the volume with 25 blocks that you adjust with the arrow keys. The 10 is just for mapping blocks 0..25 to volume 0..250